src/app/app.component.ts

```diff
@@ -1,10 +1,47 @@
 import { Component } from '@angular/core';
 
+
+
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.css']
 })
 export class AppComponent {
   title = 'app';
-}
+  toogle: boolean = true;
+  time = '';
+  state = '';
+
+  arr: any[] = [];
+
+  toggles(){
+    if(this.toogle === true) {
+      this.toogle = false;
+
+    } else {
+      this.toogle = !this.toogle;
+    }
```

Comment:
looks good. but lines 19 to 24 can be simplified as `this.toogle = !this.toogle`